src/tools/rbd/Utils.cc

```diff
@@ -422,8 +422,13 @@ int validate_snapshot_name(at::ArgumentModifier mod,
 
 int get_image_options(const boost::program_options::variables_map &vm,
 		      bool get_format, librbd::ImageOptions *opts) {
-  uint64_t order, features, stripe_unit, stripe_count, object_size;
+  uint64_t order, stripe_unit, stripe_count, object_size;
```

Comment:
Minor: order, stripe_unit, and stripe_count cause "may be used uninitialized" compiler warnings